### ISSUE TITLE
Analytics

### ISSUE THREAD

**Issue by strugee** [SeaGL/seagl.github.io#381]
But something we host ourselves on SeaGL infra lol

**Comment by strugee** [comment]
Matomo apparently has nonprofit discounts available for their cloud hosting, which is one less thing we'd have to run. Not sure how we feel hosting this data with a cloud vendor though given our privacy emphasis.

**Comment by strugee** [comment]
@altsalt what do you think of this? My feeling is that it's _probably_ okay as long as we respect Do Not Track signals, etc. and just host on Dokku/OSUOSL. I could be easily convinced that it isn't though.

One possible compromise that would be harder to convince me is too invasive is to set up something that gives us basic analytics, but doesn't use cookies.

**Comment by altsalt** [comment]
Yeah, I'm not sure if this is a tenable thing to setup with a static site. Maybe something with pretalx? I am very hesitant about the whole concept but open to some version of it.

**Comment by strugee** [comment]
What? I think you're misunderstanding. We would host on a subdomain and then include analytics code with a `<script>`. I do the same thing on strugee.net (this is an actual snippet from the ([static](https://github.com/strugee/strugee.github.com/blob/e228491cf725bf60e36a54a67e5685550e3de5eb/index.html#L117)) HTML source):

```html
      <!-- Piwik-->
      <!-- Note that this Piwik instance is self-hosted and so doesn't leak metadata. Also, critically, it respects Do Not Track.-->
      <script>
        var _paq = _paq || [];
        _paq.push(['trackPageView']);
        _paq.push(['enableLinkTracking']);
        (function() {
        var u="//piwik.strugee.net/";
        _paq.push(['setTrackerUrl', u+'piwik.php']);
        _paq.push(['setSiteId', 1]);
        var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
        g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
        })();
      </script>
      <noscript>
        <p><img src="//piwik.strugee.net/piwik.php?idsite=1" style="border:0;" alt=""></p>
      </noscript>
      <!-- End Piwik Code-->
```
